discordbot/stocks/technical_analysis/ta_menu.py

```diff
@@ -0,0 +1,916 @@
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await adosc_command(ctx, ticker, open, fast, slow, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.obv")
+    async def obv(self, ctx: discord.ext.commands.Context, ticker="", start="", end=""):
+        """Displays chart with obv of a given stock and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await obv_command(ctx, ticker, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.fib")
+    async def fib(self, ctx: discord.ext.commands.Context, ticker="", start="", end=""):
+        """Displays chart with fib of a given stock and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        start:
+            date (in date format for start date)
+        end:
+            date (in date format for end date)
+        """
+
+        await fib_command(ctx, ticker, start, end)
+
+    @discord.ext.commands.command(name="stocks.ta.view")
+    async def view(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays an image of a given stocks data from Finviz and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await view_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta.summary")
+    async def summary(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays a summary of a given stocks ta from FinBrain and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await summary_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta.recom")
+    async def recom(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Displays a recommendation of a given stocks ta from Tradingview and sends it
+
+        Parameters
+        -----------
+        ticker: str
+            ticker, -h or help
+        """
+
+        await recom_command(ctx, ticker)
+
+    @discord.ext.commands.command(name="stocks.ta")
+    async def ta(self, ctx: discord.ext.commands.Context, ticker=""):
+        """Stocks Context - Shows Technical Analysis Menu
+
+        Returns
+        -------
+        Sends a message to the discord user with the commands from the stocks.ta context.
+        The user can then select a reaction to trigger a command.
+        """
+
+        if cfg.DEBUG:
+            print(f"!stocks.ta {ticker}")
+
+        cols_temp = []
+        cols = []
+
+        if ticker != "":
+            stock = helpers.load(ticker, datetime.now() - timedelta(days=365))
+            if stock.empty:
```

Comment:
To make it simpler you can check here for stock.empty or ticker == "", and that way there's no need for an extra else